src/config/config.js

```diff
@@ -210,5 +210,15 @@ export default {
    * @memberof module:config
    * @property {Object} indutny_elliptic_fetch_options Options object to pass to `fetch` when loading the indutny/elliptic library. Only has an effect if `config.external_indutny_elliptic` is true.
    */
-  indutny_elliptic_fetch_options: {}
+  indutny_elliptic_fetch_options: {},
+  /**
+   * @memberof module:config
+   * @property {Array<Integer>} reject_hash_algorithms Reject insecure hash algorithm {@link module:enums.hash}
+   */
+  reject_hash_algorithms: new global.Set([enums.hash.md5, enums.hash.ripemd]),
```

Comment:
Why Use global.Set here? Seems inconsistent with how other config values are set.

Reply:
if other config variable is reject message hash, than may be this one should be "reject_key_hash_algorithms"?

Reply:
> Why Use global.Set here? Seems inconsistent with how other config values are set.

I initially made an array, but I switched to a Set so that you can easily do:

```js
openpgp.config.reject_message_hash_algorithms.delete(openpgp.enums.hash.sha1);
```

The only other array in config.js is `known_notations`, there I think it's less necessary to be able to easily remove them?

The `global.Set` instead of `Set` is because of https://github.com/openpgpjs/openpgpjs/commit/7bb8b69044f9b3309b5f31f0ba02d03de43276eb.

> if other config variable is reject message hash, than may be this one should be "reject_key_hash_algorithms"?

Maybe, but we might in the future also want to use it for other places where hashes are used (like S2K). I think long term the other config (`reject_message_hash_algorithms`) should eventually be removed again and we'll keep just this one.